src/meta/src/hummock/manager/mod.rs

```diff
@@ -1137,7 +1160,10 @@ impl HummockManager {
     #[named]
     pub async fn report_compact_task_impl(
         &self,
-        compact_task: &mut CompactTask,
+        task_id: u64,
+        compact_task: Option<CompactTask>,
```

Comment:
As `compact_task` can be Some only when it's trivial move, what about rename it to something like `trivial_move_compact_task`, to emphasize the fact?
Otherwise, I have to think about it before understanding why we don't need to remove it from `compact_task_assignment` subsequently.

Reply:
This parameter, which may be used in testing at the moment, may be ambiguous when renamed directly.

I'll add some comments to explain this parameter and why it doesn't appear in compact_task_assignment